layouts/highlights.html

```diff
@@ -12,6 +12,11 @@
             <p>Check out a number of <a href="http://wiki.hl7.org/index.php?title=Open_Source_FHIR_implementations">Open Source FHIR Implementations</a>.</p>
         </li>
         <li class="highlight-module">
+            <a href="/faq/"><span class="mega-octicon octicon-question"></span></a>
+            <h2><a href="/faq/">FHIR Developer FAQ</a></h2>
```

Comment:
Probably want this title to just be "FAQ" since it could eventually link to an FAQ page with other FAQ sections besides FHIR Developer FAQs